test/unit/test_blas_tile.cpp

```diff
@@ -50,6 +50,7 @@ std::vector<std::tuple<SizeType, SizeType, SizeType, SizeType, SizeType, SizeTyp
     {0, 5, 13, 1, 0, 1}, {7, 0, 4, 1, 2, 0},    {3, 11, 0, 0, 1, 0},  // one 0 size
     {1, 1, 1, 0, 3, 0},  {1, 12, 1, 1, 0, 7},   {17, 12, 16, 1, 3, 0}, {11, 23, 8, 0, 3, 4},
     {6, 9, 12, 1, 1, 1}, {32, 32, 32, 0, 0, 0}, {32, 32, 32, 4, 5, 7},
+    {128, 128, 128, 0, 0, 0},
```

Comment:
I'm going move this in and independent PR, because probably it would need a bit more attention (talking with @rasolca, it may be also useful to re-evaluate how we carry on tests about precision)

Reply:
I've attempted to add back the 128 blocksizes here which I think will still be fine for most tests in terms of precision. What isn't fine I can revert and will be handled in #557.